server/views/components/meta-unit/meta-unit.njk

```diff
@@ -0,0 +1,35 @@
+<div>
+  <{{model.headingLevel or 'h2'}} class="
+    {{ {s:'HNM5' , m:'HNM4', l:'HNM4'} | fontClasses }}
+    {{ {s:0, m:0, l:0, xl: 0} | spacingClasses({margin: ['top']}) }}
+    {{ {s:1, m:1, l:1, xl: 1} | spacingClasses({margin: ['bottom']}) }}">
+   {{ model.headingText }}
+  </{{model.headingLevel or 'h2'}}>
+  {% if model.content | isString %}
+    <p class="{{ {s:'HNL5' , m:'HNL4', l:'HNL4', xl:'HNL4'} | fontClasses }} {{ {s:1, m:1, l:1, xl: 1} | spacingClasses({margin: ['bottom']}) }}">{{model.content}}</p>
+  {% else %}
+    {% for item in model.content %}
+      {% if item.datetime %}
+        {% if loop.first %}
```

Comment:
We don't we just wrap the loop in the tag?
```twig
{% if model.content.length > 0 %}
<p>
  {% for item in model.content %}
</p>
{% .endif %}
```

It seems a little more readable.